publish/publish.proj

```diff
@@ -222,6 +222,16 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyPackagesForSigning">
```

Comment:
The copying actually isn't needed any longer. Instead you should just make DownloadDirectory and PublishDirectory above the same.